indico/htdocs/js/indico/modules/registration/form/tpls/fieldpicker.tpl.html

```diff
@@ -71,4 +71,10 @@
         <i class="icon-mail"></i>
         <span>{{ "Email" | i18n }}</span>
     </a>
+
+    <a ng-click="actions.openAddField(section, 'accommodation')"
+        class="i-big-button">
+        <i class="icon-home"></i>
+        <span>{{ "Accommodation" | i18n }}</span>
```

Comment:
At the moment it doesn't look very nice because of how many field types we currently have.